src/components/KonnectorInstall.jsx

```diff
@@ -9,9 +9,13 @@ import LegacyKonnectorInstall from 'components/LegacyKonnectorInstall'
 import { getKonnector } from 'ducks/konnectors'
 import styles from 'styles/konnectorInstall'
 
-export class KonnectorInstall extends PureComponent {
+export class KonnectorInstall extends Component {
```

Comment:
Why the change here ? Does the component rely on something external to props/state ?

Reply:
We have some complex objects from props here, and if I read correctly:

> When props or state changes, PureComponent will do a shallow comparison on both props and state.

I don't think it's safe to do an only shallow comparison for state/props here so

Reply:
OK